beans/description.js

```diff
@@ -0,0 +1,64 @@
+'use srtict';
+
+/**
+ * Enumeration of all available description types.
+ * @enum {String}
+ */
+var TYPES = {
+    TEXT: 'text',
+    HTML: 'html',
+    MARKDOWN: 'markdown'
+};
+
+function isAvailableType(type) {
+    for (var key in TYPES) {
+        if (TYPES.hasOwnProperty(key)) {
+            if (TYPES[key] === type) {
+                return true;
+            }
+        }
+    }
+
+    return false;
```

Comment:
It can be simpler

```js
function isAvailableType(type) {
  return Object.keys(TYPES).some(function(key) {
     return TYPES[key] === type;
  });
}
```